.cirrus.yml

```diff
@@ -0,0 +1,106 @@
+# Reference:
+#   - https://cirrus-ci.org/guide/writing-tasks/
+#   - https://cirrus-ci.org/guide/tips-and-tricks/#sharing-configuration-between-tasks
+#   - https://cirrus-ci.org/guide/linux/
+#   - https://cirrus-ci.org/guide/macOS/
+#   - https://cirrus-ci.org/guide/windows/
+#   - https://hub.docker.com/_/gcc/
+#   - https://hub.docker.com/_/python/
+
+#
+# Global defaults.
+#
+container:
+  image: python:3.8
+  cpu: 2
+  memory: 4G
+
+env:
+  # Maximum cache period (in weeks) before forcing a new cache upload.
+  CACHE_PERIOD: "2"
+  # Increment the build number to force new conda cache upload.
+  CONDA_CACHE_BUILD: "98"
+  # Increment the build number to force new nox cache upload.
+  NOX_CACHE_BUILD: "98"
+  # Increment the build number to force new pip cache upload.
+  PIP_CACHE_BUILD: "98"
+  # Pip package to be upgraded/installed.
+  PIP_CACHE_PACKAGES: "pip setuptools wheel nox pyyaml"
+  # Base directory for the iris-test-data.
+  IRIS_TEST_DATA_DIR: ${HOME}/iris-test-data
+  # Git commit hash for iris test data.
+  IRIS_TEST_DATA_REF: "2.0.0"
+
+#
+# Linting
+#
+lint_task:
+  auto_cancellation: true
+  allow_failures: true
+  name: "${CIRRUS_OS}: flake8 and black"
+  pip_cache:
+    folder: ~/.cache/pip
+    fingerprint_script:
+      - echo "${CIRRUS_TASK_NAME}"
+      - echo "$(date +%Y).$(expr $(date +%U) / ${CACHE_PERIOD}):${PIP_CACHE_BUILD} ${PIP_CACHE_PACKAGES}"
+  lint_script:
+    - pip list
+    - python -m pip install --retries 3 --upgrade ${PIP_CACHE_PACKAGES}
+    - pip list
+    - "nox --session flake8 || :"
+    - "nox --session black || :"
```

Comment:
@jamesp I'd be more in favour of commenting out the entire `lint_task` and raise two separate issues to request work to make the `iris-grib` code base `flake8` compliant and `black` compliant.

Reply:
will do 👍